tutorials/Python/07_FileHandling/FileHandling.ipynb

```diff
@@ -11,28 +11,38 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "One important aspect of using Python is being able to read and write files. To do this you use the *open* command:\n",
-    "*open*(*filename-in-quotes*, *mode*)\n",
+    "One important aspect of using Python is being able to read and write files. To do this you use the `open` command:\n",
     "\n",
-    "In place of *mode* you can specify:\n",
-    "- 'r', to open a file you want to read\n",
-    "- 'w', to open a file that you want to write (starting a new file).\n",
-    "- 'a', to open a file that you want to write (appending to an existing file).\n",
+    "```\n",
+    "open(filename-in-quotes, mode)\n",
+    "```\n",
     "\n",
-    "So, to open a file with filename file1.txt in 'read' mode:\n",
+    "In place of `mode` you can specify:\n",
+    "- `\"r\"`, to open a file you want to read\n",
+    "- `\"w\"`, to open a file that you want to write (starting a new file).\n",
+    "- `\"a\"`, to open a file that you want to write (appending to an existing file).\n",
```

Comment:
Maybe change this to "add to a file", I'm just not sure how much people know what append means if they don't have a programming background?

Reply:
What about
> `"a"`, to open a file that you want to append to (adding to an existing file)